### ISSUE TITLE
Frame Background not working on Windows

### ISSUE THREAD

**Issue by Naryorill** [dotnet/maui#5490]
### Description

The Background property of a frame doesn't work on Windows. It works on Android.

### Steps to Reproduce

1) Create a new MAUI project

2) In MainPage.xaml, replace the Label on Row 0 with this:
```
      <Frame Grid.Row="0"
             WidthRequest="100"
             HeightRequest="100"
             BorderColor="Red"
             Background="Purple"/>
```

3) Start on Windows, you'll only see a red border.

4) Start on Android, you'll see a purple box with a red border as expected

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows Version	10.0.19044 Build 19044

**Comment by jfversluis** [comment]
Does it work if you use `BackgroundColor` instead?

**Comment by Naryorill** [comment]
Yes, BackgroundColor works, but that doesn't support brushes. Trying out the samples in the [documentation of brushes](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/brushes/solidcolor) is how I found this issue.

**Comment by jfversluis** [comment]
Gotcha, just wanted to give you a workaround in case you need it :) 

I have verified it reproduces. The code in the initial comment works on Android, but on Windows it doesn't show the background color. Using `BackgroundColor` on Windows does show an actual color.
